handbook/company/senior-software-backend-engineer.md

```diff
@@ -0,0 +1,70 @@
+
+tldr; Fleet Device Management Inc. is a [recently-funded](https://techcrunch.com/2022/04/28/fleet-nabs-20m-to-enable-enterprises-to-manage-their-devices/) Series A startup founded and backed by the same people who created osquery, the leading open source security agent. Today, osquery is installed on millions of laptops and servers, and it is especially popular with [enterprise IT and security teams](https://www.linuxfoundation.org/press/press-release/the-linux-foundation-announces-intent-to-form-new-foundation-to-support-osquery-community).
+
+
+### Your primary responsibilities 🔍
+
+As a Senior Software Engineer at Fleet, you will get the chance to…
+
+In your first 90 days:
+
+- 🧑‍🔬 Design, develop, test, and maintain a state-of-the-art Golang application that includes robust APIs to support mobile and desktop clients.
+- 🛠️ Write code and tests, build prototypes, resolve issues, and profile and analyze bottlenecks.
+- 💭 Manage and optimize scalable distributed systems in the cloud.
+- 🤝 Collaborate closely with product managers to understand requirements and translate them into actionable specifications.
+- 🚀 Actively participate in all engineering scrum meetings, including sprint planning, daily standups, sprint demos, sprint retrospectives, and estimation sessions.
+- 🌟 Contribute to the overall success of the [customer cxperience (CX)](https://fleetdm.com/handbook/company/development-groups#customer-experience-group) product group by ensuring users receive valuable new features.
+
+### Are you our new team member? 🧑‍🚀
+
+If most of these qualities sound like you, we would love to chat and see if we're a good fit.
+
+### You "get it":
+
+- 🦉 Translate requirements into well-designed and functional software.
+- 🤝 Communicate regularly with stakeholders, project managers, quality assurance teams, and other developers regarding progress on long-term technology roadmap.
+- 🧪 Collaborate with QA team for testing software features.
+- 🏃‍♂️ Familiarity with agile development processes and scrum methodologies.
+- 🛠️ Produce quality code, raising the bar for team performance and speed.
+- 📖 Mentor junior team members.
+
+### You can "walk the walk":
+
+- 🤝 Collaboration: You work best in a participatory, team-based environment.
+- 🚀 Prototype-first: You embrace speed and failure as we iterate towards the right solution. You have hands-on experience in creating low and high fidelity prototypes. You’re comfortable accepting suboptimal designs in favor of iteration.
+- 🧬 Simplicity: You love complex questions and use your work to simplify that complexity for users.
+- 🛠️ Technical: You understand the software development processes. You understand that software quality matters.
+- 🟣 Openness: You are flexible and open to new ideas and ways of working.
+- ➕ Bonus: Cybersecurity or IT background.
+
+### You can "talk the talk":
+
+- 📖 Bachelor’s degree (or equivalent) in computer science
```

Comment:
```suggestion
```

Reply:
We don't list this as a requirement to avoid discouraging quality engineers without degrees.